README.md

```diff
@@ -1,24 +1,24 @@
-# README
+# Contact.me
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Keep track of your contacts, all in one place, anytime, anywhere!
 
-Things you may want to cover:
+## Technologies
 
-* Ruby version
+- Ruby on Rails
+- React.js
+- Materialize CSS
 
-* System dependencies
+## Local Development
 
-* Configuration
+- Install npm dependencies bundle install
+- To run the app: rails s
+- To run the tests run: rspec
 
-* Database creation
+## Contributing
 
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+1. Fork this repositry to your account.
+2. Clone your repository: git clone git@github.com:andela-oduyile/contact.me.git
+3. Create your feature branch: git checkout -b new-feature
+4. Commit your changes: git commit -m "did something"
+5. Push to the remote branch: git push origin new-feature
+6. Open a pull request.
```

Comment:
Update last line to `Open a pull request, ensure that the "base fork" points to the correct repository and branch.`

Reply:
Done